<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сбор данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="gbInput">
      <property name="title">
       <string>Ввод</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbCOMPort">
         <property name="text">
          <string>COM-порт:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbCOMPort">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>COM1</string>
         </property>
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Файл:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="btFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbOutput">
      <property name="title">
       <string>Вывод</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QToolButton" name="btVectors">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Матрица:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Векторы:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edMatrix">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>matrix.txt</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="btMatrix">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edVectors">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>vectors.txt</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="btGo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="tbLog">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Журнал сбора данных.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btGo</tabstop>
  <tabstop>rbCOMPort</tabstop>
  <tabstop>cbCOMPort</tabstop>
  <tabstop>rbFile</tabstop>
  <tabstop>edFile</tabstop>
  <tabstop>btFile</tabstop>
  <tabstop>edVectors</tabstop>
  <tabstop>btVectors</tabstop>
  <tabstop>edMatrix</tabstop>
  <tabstop>btMatrix</tabstop>
  <tabstop>tbLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
